examples/michael_schumacher_problem/main_michael_schumacher.m

```diff
@@ -150,9 +153,16 @@
 % intitial guess for fun
 alpha = [atan2(diff(yy),diff(xx)),0];
 x_guess = vertcat(xx,yy, zeros(2,problem_options.N_stages), alpha);
+
+active_set_guess = nosnoc.activeset.Pss({[1,2]},'times', [problem_options.T]);
+solver_options.mpecopt.initialization_strategy = 'TakeProvidedActiveSet';
 %% Solve
 ocp_solver = nosnoc.ocp.Solver(model, problem_options, solver_options);
-ocp_solver.solve();
+
+ocp_solver.set_initial_active_set(active_set_guess);
+[IG,IH,I00] = ocp_solver.discrete_time_problem.process_active_set(active_set_guess);
+
+ocp_solver.solve('mpecopt', IG=IG, IH=IH);
 fprintf('Objective values is: %2.4f \n', ocp_solver.get_objective());
 %% plot
 plot_results_ms(model, problem_options, ocp_solver,...
```

Comment:
a very nice option would be, if the active set method is used, to plot all BNLP iterates -  to see the evolution of active set changes. 

ocp_solver.stats.iter.X_outer 
I believe, but I am not sure how to extract states and controls from them.

Reply:
This can be done by setting `ocp_solver.discrete_time_problem.w.res = ocp_solver_states.iter.X_outer{i}`, assuming they are returned as the primal variables for the unlifted, original form.

I believe we had done this earlier somewhere, and I can quickly implement it again if you wish.